src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -34,6 +35,26 @@
 )
 
 
+class Aggressiveness(Enum):
+    """
+    Enum for aggressiveness of the bot
+    """
+
+    aggressive = "aggressive"
+    moderate = "moderate"
+    friendly = "friendly"
+
+
+class Aggressiveness(Enum):
+    """
+    Enum for aggressiveness of the bot
+    """
+
+    aggressive = "aggressive"
+    moderate = "moderate"
+    friendly = "friendly"
```

Comment:
You accidentally duplicated this again.

Reply:
fixed! Sorry I'm not sure why it does this. They seem to appear from rebasing but I never resolved any merge conflicts in this area of the code.